controllers/apply/forms/reaching-communities-form.js

```diff
@@ -100,70 +100,92 @@ formModel.registerStep({
                             }
                         });
                     }
+                },
+                {
+                    type: 'text',
+                    name: 'project-location',
+                    label: "In your own words, describe the location(s) that you'll be running your project(s) in",
+                    placeholder: 'eg. "Newtown community centre" or "Alfreton, Derby and Ripley"',
```

Comment:
It's a little odd to have a Newtown as an example when this programme is for England only.